### DISCUSSION TITLE
Read Documentation !Important

### DISCUSSION THREAD

**Discussion by ghost** [IlluxDev/Oxide#1] [General]
@EmeraldMike @DarkMalwre @Xcyth and @DarkMalwre you may prefer to start reading the documentation for Oxide WebSocket at trying out some things on release, if you want to try out code right now, after school, DM me for a quick setup document.

The docs url is listed on this page
https://github.com/illuxdev/oxide

**Comment by DarkMalwre** [comment]
Alr I'll read it

**Reply by ghost** [reply]
Nice! Good Luck!

**Reply by DarkMalwre** [reply]
speedrun time lmao

**Reply by ghost** [reply]
Could you also tell mike to get on it? He doesnt seem to have noticed

**Reply by NekaouMike** [reply]
BRUH GITHUB HAS SENT 8 EMAILS AND COUNTING!
Also look on discord.
I NOTICED!

**Reply by DarkMalwre** [reply]
LOL

**Reply by DarkMalwre** [reply]
XFaon can't get on discord in school btw

**Reply by NekaouMike** [reply]
oh yeah righttt.
The emails are still anonying

**Reply by Xcyth** [reply]
i am about to commit death.

**Reply by NekaouMike** [reply]
Why?

**Reply by Xcyth** [reply]
the fucking emails

**Reply by DarkMalwre** [reply]
oh lmao

**Reply by ghost** [reply]
> BRUH GITHUB HAS SENT 8 EMAILS AND COUNTING! Also look on discord. I NOTICED!

I'm at school mike, can't use phones at this time

**Reply by DarkMalwre** [reply]
use your laptop

**Reply by DarkMalwre** [reply]
https://discord.com/app

**Reply by NekaouMike** [reply]
lol

**Reply by NekaouMike** [reply]
school wifi go discord bad bad

**Reply by ghost** [reply]
> use your laptop

my desk is right infront of the teacher BRUH

**Comment by ghost** [comment]
Ill try
